pkg/storaged/storage.scss

```diff
@@ -408,3 +408,7 @@ td.storage-action {
 .pf-c-description-list .pf-c-progress {
     max-width: 30ch;
 }
+
```

Comment:
this essentially overrides https://github.com/cockpit-project/cockpit/blob/main/pkg/lib/patternfly/patternfly-4-overrides.scss#L128 

We need to fix that rule instead.

Reply:
The way nested forms are implemented in the storage page within dialog.jsx is wrong in the first place. The nested element should just be children of the FormGroup that they belong to.
Like here: https://codesandbox.io/s/f3gfq